audio-search/flows/query.yml

```diff
@@ -1,29 +1,36 @@
 !Flow
+version: '1'
 with:
   read_only: true
   rest_api: true
   port_expose: $JINA_PORT
 pods:
-  crafter:
-    uses: pods/craft.yml
-    parallel: $PARALLEL
-  encoder:
+  - name: segmenter
+    uses: pods/segment.yml
+    shards: $SHARDS
+    timeout_ready: 600000
+    show_exc_info: true
+  - name: encoder
     uses: pods/encode.yml
-    parallel: $PARALLEL
+    shards: $SHARDS
     timeout_ready: 600000
     read_only: true
-  vec_idx:
+    show_exc_info: true
+  - name: vec_idx
     uses: pods/vec.yml
     shards: $SHARDS
-    separated_workspace: true
     polling: all
     uses_reducing: _merge_all
```

Comment:
This uses reducing to be changed by another pod yaml, u can inspire from `multires example`

Reply:
Thanks! 😄